src/main/java/entity/Portfolio.java

```diff
@@ -0,0 +1,29 @@
+package entity;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Portfolio {
+
+    private final Map<String, UserStock> stocks;
+
+    public Portfolio() {
+        this.stocks = new HashMap<>();
+    }
+
```

Comment:
We may also add a `Portfolio` constructor that takes an existing Map<String, int>, so that when we need to construct the `Portfolio` for an existing user, we can initialize it with the user's stock data. This comment also applies for `TransactionHistory`.